caas/kubernetes/provider/namespaces.go

```diff
@@ -111,22 +106,26 @@ func (k *kubernetesClient) GetCurrentNamespace() string {
 
 func (k *kubernetesClient) ensureNamespaceAnnotations(ns *core.Namespace) error {
 	annotations := k8sannotations.New(ns.GetAnnotations()).Merge(k.annotations)
-	// check required keys are set: utils.AnnotationControllerUUIDKey, utils.AnnotationModelUUIDKey.
-	if err := annotations.CheckKeysNonEmpty(requireAnnotationsForNameSpace...); err != nil {
+	err := annotations.CheckKeysNonEmpty(utils.AnnotationControllerUUIDKey(false), utils.AnnotationModelUUIDKey(false))
```

Comment:
so its the same check, just without the global var, right?

Reply:
Yes. Only used in this place, and placing inline removes the need for the comment.